src/assistants/openai/openai_assistant.rs

```diff
@@ -75,6 +77,25 @@ impl OpenAIAssistant {
         self
     }
 
+    ///
+    /// This method can be used to define the model temperature used by the Assistant
+    /// This method accepts % target of the acceptable range for the model
+    ///
+    pub fn temperature(mut self, temp_target: u32) -> Self {
```

Comment:
I gave users 2 options: either pass the temperature unchecked or use a safe way of setting it within the accepted range.